### PR TITLE
Explain how to use React Developer Tools

### PR DESCRIPTION
**Pull request by gaearon**
Existing instructions are a bit misleading, as Chrome extension is not supposed to work, but there is still a supported way to run them.

### CODE REVIEW

File: docs/Debugging.md [hunk 1]
```diff
@@ -71,6 +73,20 @@ The debugger will receive a list of all project roots, separated by a space. For
...
+
+```js
+npm install -g react-devtools
+```
+
+Now run `react-devtools` from the terminal to launch the standalone DevTools app.
+
+It should connect to your simulator within a few seconds.
+
+> Note: if you prefer to avoid global installations, alternatively you can run `npm install --save-dev react-devtools` and add `"react-devtools": "react-devtools"` to the `scripts` section in your `package.json`. Then you could run `npm run react-devtools` from your project folder.
```
Comment (kittens) [inline docs/Debugging.md]: Not sure if you care but with Yarn you can run any script in `node_modules/.bin` by just running `yarn <script-name>`. So for example with Yarn all you need to do is `yarn add react-devtools; yarn react-devtools`.

**Comment by gaearon** [comment]
Thanks, added.

File: docs/Debugging.md [hunk 2]
```diff
@@ -71,6 +73,20 @@ The debugger will receive a list of all project roots, separated by a space. For
 
 > Custom debugger commands executed this way should be short-lived processes, and they shouldn't produce more than 200 kilobytes of output.
 
+## React Developer Tools
+
+With React Native 0.43 or higher, you can use [the standalone version of React Developer Tools](https://github.com/facebook/react-devtools/tree/master/packages/react-devtools) to debug the React component hierarchy. To use it, install the `react-devtools` package globally:
+
+```js
```
Comment (timbuckley) [inline docs/Debugging.md]: Doesn't matter but the markdown syntax here should be `sh` or `bash` (or nothing at all), not `js` :-)

**Comment by spicyj** [comment]
Can we add a note in the RN Chrome debugging page itself too?

File: docs/Debugging.md [hunk 3]
```diff
@@ -71,6 +73,24 @@ The debugger will receive a list of all project roots, separated by a space. For
...
+
+Now run `react-devtools` from the terminal to launch the standalone DevTools app:
+
+```
+react-devtools
+```
+
+It should connect to your simulator within a few seconds.
+
+> Note: if you prefer to avoid global installations, you can add `react-devtools` as a project dependency. With Yarn, you can run `yarn add react-devtools`, and then run `yarn react-devtools` from your project folder to open the DevTools. With npm, you can run `npm install --save react-devtools`, add `"react-devtools": "react-devtools"` to the `scripts` section in your `package.json`, and then run `npm run react-devtools` from your project folder to open the DevTools.
```
Comment (gaearon) [inline docs/Debugging.md]: Does it matter though? The distinction is mostly important for Node apps.

Comment (satya164) [inline docs/Debugging.md]: Matters when you're publishing a library

**Comment by gaearon** [comment]
@spicyj I added it to the `debugger.html`:

screen shot 2017-05-01 at 20 03 41

**Comment by ericnakagawa** [comment]
Great work! Standalone React Dev-tools is helpful for folks who don't want to complicate their tabs or who aren't using Chrome as their primary browser.

**Comment by gaearon** [comment]
>Standalone React Dev-tools is helpful for folks who don't want to complicate their tabs or who aren't using Chrome as their primary browser.

This not quite right—Chrome plugin **does not** work in RN at all. It hasn’t for a long time. It is not possible to fix it to work with RN, so the standalone app is a replacement for that.

**Pull request closed without merging**